linux-tools/prelink/prelink.sh

```diff
@@ -182,9 +190,13 @@ function test02()
 		tc_fail "prelink failed on bogus library dependency" || return
 
 	fi
-
-	grep -q "^`echo $PRELINK | sed 's/ .*$/: .*has a dependency cycle/'`" $BINS.log
-	tc_fail_if_bad $? "prelink failed on cycle1" || return
+	
+	# cycle1lib1.so libs are not part of Ubuntu, so excluding this test
+	grep -i "ubuntu" /etc/*-release >/dev/null 2>&1
```

Comment:
cycle1lib1.so libs are created for test purpose ? If so this cannot be built for ubuntu ?